model/custom_status.go

```diff
@@ -15,9 +16,21 @@ const (
 	MaxRecentCustomStatuses  = 5
 )
 
+var ValidCustomStatusDuration = map[string]bool{
```

Comment:
nit: This doesn't feel like something we should export. What if we just inlined a switch statement into the corresponding method instead?

Reply:
We could just lower case it to avoid exporting it. Our tendency to capitalize all constants does mean we're exporting a lot.